python-package/xgboost/core.py

```diff
@@ -1244,55 +1261,54 @@ def feature_names(self, feature_names: Optional[FeatureNames]) -> None:
 
```

Comment:
Can you move this docstring to the getter? The docstring in the setter isn't visible in the generated doc. See https://stackoverflow.com/a/16025754

Reply:
Done.